src/main.lua

```diff
@@ -2133,4 +2141,252 @@ end)
 
 -- END READ HANDLERS
 
+-- AUCTION HANDLER
+addEventingHandler("releaseName", utils.hasMatchingTag("Action", ActionMap.ReleaseName), function(msg)
+	-- validate the name and process id exist, then create the auction using the auction function
+	local name = msg.Tags.Name
+	local processId = msg.From
+	local record = arns.getRecord(name)
+	local initiator = msg.Tags.Initiator
```

Comment:
Can we be getting this from a Forwarded-For tag or similar that's provided by the MU during message forwarding?